properties_pane/field_level/fieldLevelConfig.json

```diff
@@ -142,7 +142,40 @@ making sure that you maintain a proper JSON format.
 				"enableForReference": true,
 				"propertyType": "checkbox"
 			},
-			"primaryKey",
+			{
+				"fieldName": "Primary Key",
```

Comment:
propertyName, propertyKeyword etc. This part doesn't make any sense for Glue
```
				{
						"key": "refType",
						"value": "collectionReference"
					},
					{
						"level": "root",
						"key": "viewOn",
						"exist": true
					},
					{
						"type": "not",
						"values": {
							"level": "parent",
							"key": "type",
							"value": "object"
						}
					},
```

changeAction is not needed,